intelmq/lib/bot.py

```diff
@@ -199,6 +199,8 @@ def handle_sighup_signal_threading(signum: int,
             self.__load_harmonization_configuration()
 
             self._parse_common_parameters()
+
```

Comment:
`super().__init__()` has to be initialized before the `self.init()`, because Mixins have to be available before the init is called.

Reply:
Thanks, sounds logical

Reply:
Fixed